guide/17-geocoding-guides/part6_custom_geocoders.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<p><code>Geocoders</code> have several properties that can be accessed through <code>geocoder.properties</code>. You can find the basic walk through of these properties in <a href="https://app.reviewnb.com/Esri/arcgis-python-api/pull/830/part1_what_is_geocoding.ipynb#Properties-of-the-geocoder-class" target="_blank">Part 1</a>. Here, let's look at some other details of geocoder properties.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>